Content.Client/_CM14/Scoping/ScopeSystem.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Numerics;
+using Content.Client.Movement.Systems;
+using Content.Shared._CM14.Scoping;
+using Content.Shared.Camera;
+
+namespace Content.Client._CM14.Scoping;
+
+public sealed class ScopeSystem : SharedScopeSystem
+{
+    [Dependency] private readonly ContentEyeSystem _eye = default!;
+
+    protected override void StartScopingCamera(EntityUid user, ScopeComponent scopeComponent)
+    {
+    }
+
+    protected override void StopScopingCamera(EntityUid user, ScopeComponent scopeComponent)
+    {
+        if (TryComp<CameraRecoilComponent>(user, out var cameraRecoilComponent))
```

Comment:
This is mostly a style thing but it does matter in other APIs, we usually put the type next to out, instead of as a generic parameter explicitly.
Feel free to change it in other cases if you want to.
```suggestion
        if (TryComp(user, out CameraRecoilComponent? cameraRecoilComponent))
```

Reply:
Sure, changed here and in other places